tests/test_replication_filter_data.py

```diff
@@ -318,5 +318,5 @@
     "landmark_897",
     "landmark_898",
     "landmark_899",
-    "landmark_900"
```

Comment:
Why in the world would this be "better"?

Reply:
These are the opinions of the community, with the idea that you don't get a choice in the style: https://github.com/psf/black#the-black-code-style

However, this is preferred because python shares the "feature" of C where two strings next to each other get concatenated, and it prevents you appending the set with `landmark901` and then accidentally having the last entry as `landmark900landmark901`:

```
foo = { "landmark1",
     ...
     "landmark900"
     "landmark901"
}
```

Perhaps the question to ask is why JSON does not allow trailing commas?